client/src/components/DragAndDrop/assets/item.module.scss

```diff
@@ -0,0 +1,115 @@
+:root{
+    --grid: 8px;
+    --grid2: 4px;
+    --borderRadius: 2px;
+    --black: #222;
+    --scrollHeight: 94%;
+    --nudeColor: #001e3c;
+    --leaving: #ffeaea;
+    --entering: #eaffeb;
+    --Height: 250px;
+}
+
+.columnParentContainer{
+  padding-bottom: 20px;
+  display: flex;
+  justify-content: center;
+  flex: auto;
+   & > div{
+    color: #fff;
+    padding: 2.5px;
+    border-radius: 10px;
+  }
+}
+
```

Comment:
I actually would like to see you use the `makeStyles` from MUI as oppposed to Sass stylesheets.